src/pages/Admin/Charity/Banking/RecipientDetails/useRecipientDetails.ts

```diff
@@ -183,9 +182,3 @@ export default function useRecipientDetails(
     updateDefaultValues,
   };
 }
-
-// Random calculation, will be handled in separate issue
-// See https://linear.app/angel-protocol/issue/AP-859/set-an-actual-calculation-to-get-the-estimate-withdrawal-amount-when
-function calculateExpectedWithdrawAmount(sourceAmount: number): number {
```

Comment:
See Andrey's comment https://linear.app/angel-protocol/issue/AP-859/set-an-actual-calculation-to-get-the-estimate-payout-amount-when#comment-bcfc2179